apps/files/src/components/FileSharingSidebar.vue

```diff
@@ -125,15 +104,23 @@ export default {
     selectedFile () {
       return this.highlightedFile
     },
-    $_ocCollaborators_users () {
-      return this.shares.filter(collaborator => {
-        return collaborator.info.share_type === '0' || collaborator.info.share_type === '6'
-      })
-    },
-    $_ocCollaborators_groups () {
-      return this.shares.filter(collaborator => {
-        return collaborator.info.share_type === '1'
-      })
+    $_ocCollaborators () {
+      return this.shares
+        .filter(collaborator => this.$_ocCollaborators_isUser(collaborator) || this.$_ocCollaborators_isGroup(collaborator))
```

Comment:
At first I thought we could get rid of the filter but then remembered that maybe link shares are also present in this structure ?

Reply:
Yes, link shares are present in the shares list.